packages/toolpad-core/src/DashboardLayout/DashboardLayout.tsx

```diff
@@ -18,7 +18,12 @@ import Toolbar from '@mui/material/Toolbar';
 import Typography from '@mui/material/Typography';
 import ExpandLessIcon from '@mui/icons-material/ExpandLess';
 import ExpandMoreIcon from '@mui/icons-material/ExpandMore';
-import { BrandingContext, Navigation, NavigationContext, NavigationPageItem } from '../AppProvider';
+import {
+  BrandingContext,
+  Navigation,
+  NavigationContext,
+  NavigationPageItem,
+} from '../AppProvider/AppProvider';
```

Comment:
Importing from `AppProvider/index` had an error in SSR as it was missing some exports from the file itself.